server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java

```diff
@@ -206,7 +210,10 @@ public void onResponse(Void unused) {
                         // and to-be-decommissioned cluster manager is no more part of Voting Configuration and no more to-be-decommission
                         // nodes can be part of Voting Config
                         listener.onResponse(new ClusterStateUpdateResponse(true));
-                        failDecommissionedNodes(clusterService.getClusterApplierService().state());
+
+                        weighAwayDecommissionedZone(state, decommissionRequest);
```

Comment:
Since the status is `DRAINING` lets call this method `drainNodesWithDecommissionedAttribute`

Reply:
Done.